### PR TITLE
feat: am perftest env changes

### PR DESCRIPTION
**Pull request by shashank-rastogi**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: k8s/perftest/common/am/accessmgmt-api.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      applicationInsightsInstrumentKey: "3331dc05-6264-4ea7-9229-c19f8ac55d3e"
+      image: hmctspublic.azurecr.io/am/accessmgmt-api:prod-647627a2
+      environment:
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
+          -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.3.1.jar
+      prometheus:
+        enabled: true
+    global:
+      environment: perftest
+      subscriptionId: "1c4f0704-a29e-403d-b719-b90c34ef14c9"
```
Comment (adusumillipraveen) [inline k8s/perftest/common/am/accessmgmt-api.yaml]: This isn't 
```suggestion
      subscriptionId: "7a4e3bd5-ae3a-4d0c-b441-2188fee3ff1c"
```

File: k8s/perftest/common/am/accessmgmt-api.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: am-accessmgmt-api
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: am-accessmgmt-api
+    version: 0.0.6
```
Comment (adusumillipraveen) [inline k8s/perftest/common/am/accessmgmt-api.yaml]: Please bump to latest in AAT as well.

Comment (shashank-rastogi) [reply k8s/perftest/common/am/accessmgmt-api.yaml]: 👍 
done

File: k8s/perftest/common/am/accessmgmt-api.yaml (hunk 1 above)
Comment (shashank-rastogi) [reply k8s/perftest/common/am/accessmgmt-api.yaml]: done

File: k8s/perftest/common/am/accessmgmt-api.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      replicas: 2
+      memoryRequests: '768Mi'
+      cpuRequests: '500m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1000m'
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "3331dc05-6264-4ea7-9229-c19f8ac55d3e"
```
Comment (adusumillipraveen) [inline k8s/perftest/common/am/accessmgmt-api.yaml]: this looks same as AAT?

Comment (shashank-rastogi) [reply k8s/perftest/common/am/accessmgmt-api.yaml]: yes it does.. i hv changed it now

**Pull request merged**